istioctl/cmd/istioctl/testdata/v1alpha3/merged-gateway.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
Should these be part of the PR? I think they're generated by a test, I _thought_ I added them to gitignore a while back...

Reply:
my bad